### PR TITLE
Update literacy

### PR DESCRIPTION
**Pull request by cmilica**

### CODE REVIEW

File: literacy_vocab.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+- Training Dataset
+- Classification
+  - Decision Tree
+  - Neural Networks
+- Test Set
+- Validation Set
+- Model Selection
+- Hyperparameters
+- Evaluation Metrics
```
Comment (agitter) [inline literacy_vocab.md]: The [performance guide](https://github.com/gitter-lab/ml-bio-workshop/blob/master/guide/performance.md) has specific evaluation-related terms as sub-section headers or in bold text.  These are the terms like `true positive`, `precision`, etc.

File: literacy_vocab.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Literacy Vocabulary 
+
+- Machine Learning
+- Dataset
+  - Samples
+  - Features
+  - Class Labels
+- Supervised/Unsupervised Learning
```
Comment (agitter) [inline literacy_vocab.md]: There are defined in the [motivation guide](https://github.com/gitter-lab/ml-bio-workshop/blob/master/guide/Motivation.md)

**Comment by agitter** [comment]
@cmilica should we find an existing machine learning glossary instead of trying to write our own?  For instance, I found https://developers.google.com/machine-learning/glossary/.  They have advanced terms but also many (or all?) of the terms we came up with.

**Comment by cmilica** [comment]
I added those as a reference to the concepts we want to focus on in the workshop. The google glossary is great. If we decide to build the reference glossary for the participants, we should use it!

**Pull request closed without merging**